guides/source/security.md

```diff
@@ -292,7 +292,13 @@ There are many other possibilities, like using a `<script>` tag to make a cross-
 
 NOTE: We can't distinguish a `<script>` tag's origin—whether it's a tag on your own site or on some other malicious site—so we must block all `<script>` across the board, even if it's actually a safe same-origin script served from your own site. In these cases, explicitly skip CSRF protection on actions that serve JavaScript meant for a `<script>` tag.
 
-To protect against all other forged requests, we introduce a _required security token_ that our site knows but other sites don't know. We include the security token in requests and verify it on the server. This is a one-liner in your application controller, and is the default for newly created Rails applications:
+To protect against all other forged requests, we introduce a _required security token_ that our site knows but other sites don't know. We include the security token in requests and verify it on the server. This can be enabled by setting `config.default_protect_from_forgery` in the [application configuration](configuring.html#configuring-action-controller):
+
+```ruby
+config.action_controller.default_protect_from_forgery = true
+``` 
+
+or by manually adding this one-liner to your application controller:
```

Comment:
```suggestion
To protect against all other forged requests, we introduce a _required security token_ that our site knows but other sites don't know. We include the security token in requests and verify it on the server. This is done automatically when `config.action_controller.default_protect_from_forgery` is set to `true`, which is the default for newly created Rails applications. You can also do it manually by adding the following to your application controller:
```

And for coherence, perhaps change the next sentence from "This will automatically include a security token" to "This will include a security token".